posts/2023/july/linguist/2023-07-13-linguist.md

```diff
@@ -13,58 +13,58 @@ image:
 lang:
 ---
 
-Browser is most used program by me and by most users of internet, so it is important for browser to be high efficient for daily routine. Altough browsers have embedded translation features and exists a lot of browser extensions for translation, all of them have bad quality and absolutely not acceptable for intensive use. This is why i've create Linguist - ultimate browser extension for translation.
+The web-browser is possibly the most used program for me and for most users of the internet, so it is important for browser to work efficiently during daily use. Altough most browsers have embedded translation features and there exists a lot of browser extensions providing translation capabalities, I have found them to, in my opinion, not good enough and unacceptable for intensive use. This is why I have created Linguist - the ultimate browser extension for translation.
 
 ![translationOfSelectedText.png](./translationOfSelectedText.png)
 
-I start development of [Linguist](https://github.com/translate-tools/linguist) in *June of 2020*, when i've tried to find solution for translation in browser, as you see, i did not. I tried really hard, but all of them had very bad quality, a [mate translate](https://gikken.co/mate-translate/) quality stood out significantly and it has good UI, however it is not have open source code, collects analytics and most of basic features are paid (even history and dictionary).
+I started development of [Linguist](https://github.com/translate-tools/linguist) in *June of 2020*, when I tried to find solution for my translation needs while using the browser but as you read above I could find that worked to my satisfaction. I found one extension - [mate translate](https://gikken.co/mate-translate/), whose quality stands out and it even has a good UI. However it is not an open source project, it collects analytics and even some of the basic features like history and dictionary are available only to paid users.
```

Comment:
> I could find

Probably it's typo, i've mean that i did not find a solution. Should we fix it to "i couldn't find"?

> as you read above

Let's not to refer readers to back to text and remove this text. It is important for me, to allow people start read from any paragraph with no context loosing and with not thinks like "should i read above to understand?".

---

> could find that worked to my satisfaction

This changes make text less aggressive compared to "I tried really hard, but all of them had very bad quality", but this is my tactic, i'm attack exists solutions and blame it - i've forced to waste time to create solution meanwhile exists about 100 translators with authors who too lazy to implement features Linguist have. My aggressive style is attempt to show this problem.

Let's leave something like "it's have very bad quality, it's garbage". We have to show the problems in software developers community